src/keystore.cpp

```diff
@@ -171,7 +171,7 @@ bool CBasicKeyStore::AddSaplingIncomingViewingKey(
 
     // Add addr -> SaplingIncomingViewing to SaplingIncomingViewingKeyMap
     mapSaplingIncomingViewingKeys[addr] = ivk;
```

Comment:
I think this still needs to be persisted in some way.

Reply:
I can't recall the history/implementation flow here. @str4d might be able to shed some light on this

Reply:
See my review comment.